### ISSUE TITLE
Toggle is not reachable by keyboard navigation

### ISSUE THREAD

**Issue by emmalearnscode** [EddieHubCommunity/good-first-issue-finder#104]
### Description

When using a keyboard to navigate through the site the tab button jumps straight from focussing on the hubber icon link to the search bar and is unable to tab to the toggle.

**Comment by bishal7679** [comment]
I would like to work on that. Can you assign me @emmalearnscode ?

**Comment by Cahllagerfeld** [comment]
See [this](https://github.com/EddieHubCommunity/good-first-issue-finder/pull/150#issuecomment-1193326453) comment for further information

**Comment by arunkumar0398** [comment]
I would like to work on this. Can you assign me @Cahllagerfeld ?

**Comment by adityaraute** [comment]
Since the PR was closed for inactivity, I suggest the assignee should be changed too
